opentracing-mock/src/test/java/io/opentracing/mock/MockSpanTest.java

```diff
@@ -26,55 +26,59 @@
     @Test
     public void testSetOperationNameAfterFinish() {
         MockTracer tracer = new MockTracer();
-        Span span = tracer.buildSpan("foo").start();
+        Span span = tracer.buildSpan("foo").startManual();
         span.finish();
 
         try {
             span.setOperationName("bar");
             Assert.fail();
         } catch (RuntimeException ex) {
+            // Fall through
```

Comment:
If we're going to comment this, please comment _why_.
I can see it's a fall through. In this case it's a unit test and the exception is the expected behaviour (given the `Assert.fail()` above it).
I'd rather rename `ex` to `expected` and add something like `assertThat("Expected exception", expected is(notNullValue()));` for documentation instead of stating the obvious in a comment.

Reply:
I'd rather not test standard JVM behavior (i.e., that exceptions caught are not null).  I can change the comment to something more descriptive though.  Perhaps "Comment to prevent empty block warning" or some such?

Reply:
What I meant is the `assertThat` could be _after_ the try-catch and the catch could be a simple assignment. Then you can lose the nondescript `fail()` and the nondescript comment.
Something like:
```java
RuntimeException exception = null;
try {
    span.setOperationName("bar");
} catch (RuntimeException rte) {
    exception = rte;
}
assertThat("Exception setting after finish", exception, notNullValue());
```

But in the end that's just a personal preference, so I'm okay either way.